emission/tests/analysisTests/modeinferTests/TestPipelineNew.py

```diff
@@ -0,0 +1,342 @@
+#Standard imports
+import unittest
+import json
+import logging
+import numpy as np
+from datetime import datetime, timedelta
+
+# Our imports
+from emission.core.get_database import get_db, get_mode_db, get_section_db
+import emission.analysis.classification.inference.mode as pipeline
+from emission.core.wrapper.user import User
+from emission.core.wrapper.client import Client
+import emission.tests.common as etc
+'''
+TODO:
+    Create some outliers and make sure they're stripped out
+
+'''
+class TestPipeline(unittest.TestCase):
+  def setUp(self):
```

Comment:
the problem with copying the file over is that I can't see only the changes. can you highlight the changes from old -> new by surrounding with something like `### porting change ###` to make it easier to review?